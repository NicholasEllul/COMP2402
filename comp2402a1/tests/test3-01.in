The FitnessGramâ„¢ Pacer Test is
a multistage aerobic capacity
test that progressively gets
more difficult as it continues.
The 20 meter pacer test will
begin in 30 seconds. Line up
at the start. The running
speed starts slowly, but
gets faster each minute
after you hear this
signal. [beep] A single
lap should be completed
each time you hear
this sound.
[ding] Remember
to run in a
straight line,
and run as long
as possible. The
second time you
fail to complete
a lap before the
sound, your test
is over. The test
will begin on the
word start.
On your mark,
get ready, start.

[Verse 1]
We're no strangers to love
You know the rules and so do I
A full commitment's what I'm thinking of
You wouldn't get this from any other guy

[Pre-Chorus]
I just wanna tell you how I'm feeling
Gotta make you understand

Chorus]
Never gonna give you up

Never gonna let you down
Never gonna run around and desert you
Never gonna make you cry
Never gonna say goodbye
Never gonna tell a lie and hurt you

[Verse 2]
We've known each other for so long
Your heart's been aching but you're too shy to say it
Inside we both know what's been going on
We know the game and we're gonna play it


[Pre-Chorus]
And if you ask me how I'm feeling
Don't tell me you're too blind to see

[Chorus]
Never gonna give you up
Never gonna let you down
Never gonna run around and desert you
Never gonna make you cry
Never gonna say goodbye
Never gonna tell a lie and hurt you
Never gonna give you up
Never gonna let you down
Never gonna run around and desert you
Never gonna make you cry
Never gonna say goodbye
Never gonna tell a lie and hurt you

[Bridge]
(Ooh give you up)
(Ooh give you up)
(Ooh) never gonna give, never gonna give
(give you up)
(Ooh) never gonna give, never gonna give
(give you up)

[Verse 3]
We've known each other for so long
Your heart's been aching but you're too shy to say it
Inside we both know what's been going on
We know the game and we're gonna play it

[Pre-Chorus]
I just wanna tell you how I'm feeling
Gotta make you understand

[Chorus]
Never gonna give you up
Never gonna let you down
Never gonna run around and desert you
Never gonna make you cry
Never gonna say goodbye
Never gonna tell a lie and hurt you
Never gonna give you up
Never gonna let you down
Never gonna run around and desert you
Never gonna make you cry
Never gonna say goodbye
Never gonna tell a lie and hurt you
Never gonna give you up
Never gonna let you down
Never gonna run around and desert you
Never gonna make you cry
Never gonna say goodbye
Never gonna tell a lie and hurt...
