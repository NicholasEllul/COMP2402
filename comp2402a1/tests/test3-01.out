begin in 30 seconds. Line up
lap should be completed
each time you hear
straight line,
We're no strangers to love

Never gonna make you cry
[Verse 2]
